src/condor_ce_jobmetrics

```diff
@@ -123,15 +157,19 @@ def main():
                 print >> sys.stderr, "Schedd", ad['Name'], "VO", vo, "totals:", info
             rrdtool.update(vo_fname, "N:%d:%d:%d:%d" % (info['Running'], info['Idle'], info['Held'], info['Jobs']))
 
-    job_results_final = []
-    for key, val in job_results.items():
-        val['DN'] = key[0]
-        val['VO'] = key[1]
-        val['VOMS'] = key[2]
-        job_results_final.append(val)
+    job_results_final = list_job_results_final(job_results)
     pilots_fname = rrd.path_with_spool(environ, "pilots")
     secure_json_write(pilots_fname, job_results_final)
 
+    gpu_job_results_final = list_job_results_final(gpu_job_results)
+    gpu_pilots_fname = rrd.path_with_spool(environ, "gpu_pilots")
+    #secure_json_write(gpu_pilots_fname, gpu_job_results_final)
+
+    # update ad with GPU totals (Jobs, Idle, Running, Held)
+    for key, val in gpu_total_results.items():
+        print "GPUPilotsTotal%s=%d" % (key, val)
+    print "- update:true"
```

Comment:
Please add a comment explaining what this does

Reply:
roger